nnvm/tests/python/compiler/test_to_relay.py

```diff
@@ -31,6 +31,23 @@ def check_model(sym, shapes, dtypes, params):
     relay_out = relay_rts.evaluate(relay_model)(*list(inputs.values()))
     np.testing.assert_allclose(nnvm_out.asnumpy(), relay_out.asnumpy())
 
+
+def test_split_concatenate():
```

Comment:
I guess this test should be in tests/python/frontend/nnvm_to_relay/test_forward.py.

Reply:
Modified